test/integration/admin/programs/sidebar/action-sidebar.js

```diff
@@ -224,6 +224,7 @@ context('program action sidebar', function() {
         return fx;
       }, '1')
       .routeProgram()
+      .routeForms()
```

Comment:
shouldn't nee this, since it's handled by the defaults

Reply:
ayep..